spec/swagger.yaml

```diff
@@ -9005,3 +9050,25 @@ definitions:
                       type: string
                       description: Watcher modification date
 
+  PublishedFilesRecentlyPublishedFilesListSuccessResponse:
+    allOf:
+      - $ref: '#/definitions/SuccessResponse'
+      - type: object
+        properties:
+          data:
+            type: object
+            properties:
+              items:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    fileUri:
+                      type: string
```

Comment:
description for properties please, fileUri, localeId, publshDate. All should have a description